zitadel/user_grant/funcs.go

```diff
@@ -125,3 +126,169 @@ func read(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagn
 	d.SetId(grant.GetId())
 	return nil
 }
+
+func readDS(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+
+	tflog.Info(ctx, "started read")
+
+	clientinfo, ok := m.(*helper.ClientInfo)
+
```

Comment:
Are this empty lines in the formatting expected?

Reply:
nope, going to remove